app/controllers/schedules_controller.rb

```diff
@@ -49,6 +53,11 @@ def new
           'You must save the Schedule first before each Task is assigned an ID.'
     flash.now[:info] = msg
     @schedule = params[:schedule].nil? ? Schedule.new : Schedule.new(schedule_params)
+
+    # This isn't quite right I don't think
```

Comment:
Instead of creating the default root_task in #new, do it in #create between the check for #valid? and the check for successful #save. Compare against the other #new methods to get an idea of what we typically do for #new vs #create.